src/FAudio_platform_sdl3.c

```diff
@@ -265,7 +265,7 @@ uint32_t FAudio_PlatformGetDeviceDetails(
 	FAudio_zero(details, sizeof(FAudioDeviceDetails));
 
 	devs = SDL_GetAudioPlaybackDevices(&devcount);
-	if (index > devcount)
+	if (index > (uint32_t)devcount)
```

Comment:
I worry about the latter being -1, so
```suggestion
	if ((int)index > devcount)
```